hetmech/matrix.py

```diff
@@ -38,6 +38,11 @@ def metaedge_to_adjacency_matrix(graph, metaedge, dtype=numpy.bool_,
             data.append(1)
     adjacency_matrix = sparse.csc_matrix((data, (row, col)), shape=shape,
                                          dtype=dtype)
+    if auto:
```

Comment:
Makes sense. So should it be build into the `dwwc` and `diffusion` functions themselves instead of into the `metaedge_to_adjacency_matrix`?